### PR TITLE
Initial commit for requirements

### PR DESCRIPTION
**Pull request by seco-plr**

### CODE REVIEW

File: application/src/main/java/ch/admin/seco/alv/service/competencecatalog/application/chfiche/ChFicheAssembler.java [hunk 1]
```diff
@@ -21,6 +21,7 @@ public static ChFicheDto toChFicheDto(ChFiche chFiche) {
                 .setTitle(chFiche.getTitle())
                 .setDescription(chFiche.getDescription())
                 .setOccupations(chFiche.getOccupations())
-                .setCompetences(chFiche.getCompetences());
+                .setCompetences(chFiche.getCompetences())
+                .setRequirementIds(chFiche.getRequirementIds());
```
Comment (emina-str) [inline application/src/main/java/ch/admin/seco/alv/service/competencecatalog/application/chfiche/ChFicheAssembler.java]: Why you don't have RequirementIds in other Create/Update ChFiche objects?
As I see in click-dummy, it's basically the same as with competences, so you should add them to the ChFicheApplicationService --> `CreateChFicheDto` and `UpdateChFicheDto` ... And also other places where it is needed.

File: infrastructure-web/src/main/resources/liquibase/changelog/20200109080000_added_requirements.xml [hunk 2]
```diff
@@ -0,0 +1,36 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.6.xsd">
+
+    <changeSet author="capl" id="20200109080000-01">
```
Comment (emina-str) [inline infrastructure-web/src/main/resources/liquibase/changelog/20200109080000_added_requirements.xml]: This is OK.
But you are missing the liquibase script for `CH_FICHE_REQUIREMENTS_IDS` table (we added it on the ChFiche). See example for Competences. (don't forget the index).

File: application/src/main/java/ch/admin/seco/alv/service/competencecatalog/application/chfiche/ChFicheAssembler.java (hunk 1 above)
Comment (seco-plr) [reply application/src/main/java/ch/admin/seco/alv/service/competencecatalog/application/chfiche/ChFicheAssembler.java]: I missed those. Thank you for pointing it out. Fixed in 8a2f8d8c47ba69122dede37fd90513908e5559fc

File: infrastructure-web/src/main/resources/liquibase/changelog/20200109080000_added_requirements.xml (hunk 2 above)
Comment (seco-plr) [reply infrastructure-web/src/main/resources/liquibase/changelog/20200109080000_added_requirements.xml]: You are right. Thank you for pointing it out. Fixed in 8a2f8d8c47ba69122dede37fd90513908e5559fc

**Pull request merged**